teradatalabs/mapr52-hive/files/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+mysql_install_db
+/usr/bin/mysqld_safe &
+sleep 20
+/usr/bin/mysqladmin -u root password 'root'
+killall mysqld
```

Comment:
@petroav hi anton mysqld is being killed because it will be restarted by supervisord.d
all the other docker containers are also doing same.
This part of code is only for setup of mysql password after which it will be started again by supervisord.d